app/views/hiring_staff/organisations/_vacancies_awaiting_feedback.html.haml

```diff
@@ -1,5 +1,5 @@
-%section.govuk-tabs__panel
-  %h2.govuk-heading-m= t("jobs.awaiting_feedback_jobs_with_count", count: vacancies.count)
+%section.govuk-tabs__panel.desktop-container-width
+  %h2.govuk-heading-m= t("jobs.awaiting_feedback_jobs_with_count", count: vacancies.count, jobs: vacancies.count == 1 ? t("jobs.job") : t("jobs.job").pluralize)
```

Comment:
Can we use `zero:..., one:..., other:...` in the translation file and lose the template logic?